### PR TITLE
Added relevant link

### PR DESCRIPTION
**Pull request by machineghost**
Added link to install instructions for non-Pop (ie. the majority of) users.

### CODE REVIEW

File: docs/pop-live-disk.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ Pop!_OS is remarkably flexible. You can run a full version of Pop!_OS from a USB
 
 A live disk is a handy tool to have around! If you are running Pop!_OS 18.04 then you have a recovery partition which is just like a live disk. You can access this by rebooting your laptop/desktop and tapping the ESC or Space key. There is a menu entry called 'Pop!_OS Recovery', choose that with the arrow keys and then press enter.
 
+Not running Pop!_OS_ yet? Take a [look here](https://support.system76.com/articles/live-disk/) for instructions on other OS's.
```
Comment (brs17) [inline docs/pop-live-disk.md]: @thomas-zimmerman, you'll need to remove the `_` after "OS" to properly display Pop!_OS. 

I might also elaborate on the second half of that sentence: "for instructions on creating a live disk" or something of that nature.

Comment (ahoneybun) [reply docs/pop-live-disk.md]: Updated.

File: docs/pop-live-disk.md [hunk 2]
```diff
@@ -13,6 +13,8 @@ Pop!_OS is remarkably flexible. You can run a full version of Pop!_OS from a USB
 
 A live disk is a handy tool to have around! If you are running Pop!_OS 18.04 then you have a recovery partition which is just like a live disk. You can access this by rebooting your laptop/desktop and tapping the ESC or Space key. There is a menu entry called 'Pop!_OS Recovery', choose that with the arrow keys and then press enter.
 
+Not running Pop!_OS yet? Take a [look here](https://support.system76.com/articles/live-disk/) for instructions on making a live disk on other OS's.
```
Comment (brs17) [inline docs/pop-live-disk.md]: Alex may be expert here but I believe the plural of OS does not have a `'`.

Comment (ahoneybun) [reply docs/pop-live-disk.md]: I believe it is OSes based on what Cass told me one time.

**Pull request merged**